.eslintignore

```diff
@@ -2,4 +2,5 @@
 **/lib/
 **/coverage/
 **/node_modules/
-docs/
\ No newline at end of file
+docs/
+.eslintrc.js
```

Comment:
https://typescript-eslint.io/linting/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file